sdk/iot/core/doc/mqtt_state_machine.md

```diff
@@ -28,12 +30,12 @@ The following aspects need to be handled by the application or convenience layer
 ## Design Decisions
 Porting requirements:
 - The target platform C99 compiler is generating reentrant code. The target platform supports a stack of several kB.
-- Our SDK relies on types such as `uint8_t` existing. If a platform doesn't already have them defined, they should be added to the application code using `typedef` statements.
+- The SDK relies on types such as `uint8_t` existing. If a platform doesn't already have them defined, they should be added to the application code using `typedef` statements.
 
 The SDK is provided only for the MQTT protocol. Support for WebSocket/WebProxy tunneling as well as TLS security is not handled by the SDK.

```

Comment:
>Support for WebSocket/WebProxy tunneling as well as TLS security is not handled by the SDK [](start = 48, length = 90)

I get that "not handled" is correct, but I'm not sure if someone new to our system will properly parse it.  They could think "this won't work in 1st place"

I think "not handled" could alternaly be said "out of scope" or "bring your own TLS/WebProxy" etc., correct?

Reply:
Terminology: There is an one & only one "Azure Embedded C SDK" which consists of a bunch of "client libraries". You can say "SDK" here because you are speaking on behalf of the entire Azure Embedded C SDK. If you want to talk about your "client Library", then you should use the term "IoT Embedded Devices Client Library"

Reply:
@JeffreyRichter @jspaith there are multiple comments regarding naming - we're still deciding on that offline. Once it's finalized we'll make the update.
Also, I believe the change will also be made to the folder structure (right now it's iot/core, why I named this "IoT Core SDK" initially).

Reply:
> I think "not handled" could alternaly be said "out of scope" or "bring your own TLS/WebProxy" etc., correct?

@jspaith I agree - I'll reformulate to state that the app-developer must handle those items instead of saying we're "not supporting". I now see how that can be interpreted the wrong way.

> Terminology: There is an one & only one "Azure Embedded C SDK" [...]

@JeffreyRichter I'll add this and you to an internal discussion about naming. I will make the changes once we've settled on a name.